### PR TITLE
fix: #401 Display logics show more fields than they need to be

### LINKED ISSUE #401: [Bug Report]  The display logics show more fields than they need to be
#### Steps to reproduce
1. Enter the 'Sales Orders' window

#### Screenshot or Gif
display-logic-error

#### Expected behavior
Only the field Warehouse should be displayed in the field group Delivery, and the fields User List 2, User List 3 and User List 4 should not be displayed in the field group Reference.

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactor
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, not the `master` branch
- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your `master` branch!
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number) fix #401 

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

## Description

Fixed evaluation of display logics that show more fields than needed

### Screenshots

 **Before this PR**
display-logic-error

**After this PR**
display-logic-solved

### CODE REVIEW

**Comment by yamelsenih** [comment]
Please resolve conflicts for merge

**Comment by EdwinBetanc0urt** [comment]
Hi @yamelsenih, the conflicts have been resolved

File: src/utils/ADempiere/evaluator.js [hunk 1]
```diff
@@ -14,27 +14,31 @@
 class evaluator {
   /**
    * Evaluate logic's
-   * @param {string} parentUuid Parent (Window / Process / Smart Browser)
+   * @param {string} parentUuid Parent (Window)
+   * @param {string} containerUuid Parent (Tab / Process / Smart Browser)
```
Comment (yamelsenih) [inline src/utils/ADempiere/evaluator.js]: This change does not have a change functionality, please remove it from changeset

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: src/utils/ADempiere/evaluator.js (hunk 1 above)
Comment (EdwinBetanc0urt) [reply src/utils/ADempiere/evaluator.js]: @yamelsenih Although the update of the comment did not affect any functionality, the change was necessary to document the update of the parameters, for those who develop and use the 'evaluator', however the change of the comment was removed on request.

**Pull request merged**